vscode/src/client/analyzerClient.ts

```diff
@@ -412,39 +430,18 @@ export class AnalyzerClient {
     return ["--config", this.getKaiConfigTomlPath()];
   }
 
-  public getNumWorkers(): number {
-    return this.config!.get("workers") as number;
-  }
-
-  public getIncidentLimit(): number {
-    return this.config!.get("incidentLimit") as number;
-  }
-
-  public getContextLines(): number {
-    return this.config!.get("contextLines") as number;
-  }
-
-  public getCodeSnipLimit(): number {
-    return this.config!.get("codeSnipLimit") as number;
-  }
+  public getRules(): string[] {
```

Comment:
Returning an array seems to create an issue [here](https://github.com/konveyor/editor-extensions/pull/126/files#diff-7bf6d907ee218343f5466cc2b00dd02fb1e52c9e918671e641a83fb1a8295d6aL139)

Reply:
@djzager - The `initilizeParams.analyzer_lsp_rules_path` where this function is being called...can that be an array of strings on the `initialize` request or does the array need to be joined before the request?

Reply:
Looks like I had gotten confused. The analyzer itself can take multiple paths, but the kai-analyzer-rpc (and kai-rpc-server by extension) only takes a single path.

I'll fix this now, once I have a tracking issue in kai for the future.